content/en/logs/guide/access-your-log-data-programmatically.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Programmatically Access Log Data Using the Logs Search API
+kind: guide
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn more about Log Explorer"
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn more about Search Syntax"
+- link: "/logs/search_syntax/"
+  tag: "Documentation"
+  text: "Learn more about syntax for Logs Search API"
+---
+
+
+## Overview
+
+Use the [Logs Search API][1] to programmatically access your log data and execute queries.
+
+The following examples are covered in this guide:
+
+* [Basic search](#basic-search)
+* [Sort by facet or timestamp](#sort-by-facet-or-timestamp)
+* [Limit the number of results retrieved](#limit-the-number-of-results-retrieved)
+* [Pagination](#pagination)
+
```

Comment:
Can we add a specific section on how to use timerange, with different examples, at least:
* using timestamps in place of dates (to make explicit if they are s, ms, us or ns timestamps)
* using relative time with the keyword `now` and `now - 1d`